scripts/deploy-backend.sh

```diff
@@ -51,6 +51,8 @@ set_cf_envs()
   "FRONTEND_BASE_URL"
   "LOGGING_LEVEL"
   "REDIS_URI"
+  "JWT_KEY"
```

Comment:
good that this is being added but not sure it will resolve the issue described in the summary unless the same value is being used in dev and staging.